src/ui/popup.js

```diff
@@ -542,6 +548,28 @@ export default class Popup extends Evented {
         applyAnchorClass(this._container, anchor, 'popup');
     }
 
+    _focusFirstElement() {
+        if (!this._container) return;
+
+        // This approach isn't covering all the quirks and cases but it should be good enough.
+        // If we would want to be really thorough we would need much more code, see e.g.:
+        // https://github.com/angular/components/blob/master/src/cdk/a11y/interactivity-checker/interactivity-checker.ts
```

Comment:
I think we probably don't need this comment in the code but it would be good to document this in the PR so that we can reference it in the future if we want to make this more robust.